src/App.js

```diff
@@ -1,5 +1,93 @@
+import { Console } from "@woowacourse/mission-utils";
+
+const CUSTOM_DELIMITER_PATTERN = /^\/\/(.)\\n/;
+
 class App {
-  async run() {}
+  async run() {
+    const input = await Console.readLineAsync(
+      "덧셈할 문자열을 입력해 주세요.\n"
+    );
+
+    let numbers = [];
+    let delimiters = [",", ":"];
+
+    if (input === "") {
+      Console.print("결과 : 0");
+      return;
+    }
+
+    const modifiedInput = this.findCustomDelimiters(input, delimiters);
+
+    const splitedInput = this.splitInput(modifiedInput, delimiters);
+
+    this.pushNumbers(splitedInput, numbers);
+
+    const sum = this.calculator(numbers);
+
+    Console.print(`결과 : ${sum}`);
+  }
+
+  isNaturalNumber(value) {
+    return value > 0 && Number.isInteger(value);
+  }
+
+  calculator(numbers) {
+    return numbers.reduce((acc, cur) => acc + cur, 0);
+  }
+
+  findCustomDelimiters(input, delimiters) {
+    let modifiedInput = input;
+
+    while (true) {
+      let match = modifiedInput.match(CUSTOM_DELIMITER_PATTERN);
```

Comment:
```suggestion
      const match = modifiedInput.match(CUSTOM_DELIMITER_PATTERN);
```
값이 변하지 않기 때문에 const로 해도 좋을 것 같습니다!